chalicelib/dcc/transformer_mapping.json

```diff
@@ -7,47 +7,28 @@
       },
       {
         "source": "metadata.json",
-        "index_field": "analysis_type",
-        "dss_field": "analysis_type"
+        "index_field": "center_name",
+        "dss_field": "core_metadata.contributor"
       },
       {
         "source": "metadata.json",
-        "dss_field": "center_name",
-        "index_field": "center_name"
+        "dss_field": "project_id",
+        "index_field": "core_metadata.project"
```

Comment:
These values show be swapped (`core_metadata.project` is a `dss_field`, and vice-versa).